aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -59,11 +62,10 @@ public void processDailyEmail(Instant scheduledFireTime) {
         }
     }
 
-    List<AggregationCommand> processAggregateEmails(Instant scheduledFireTime) {
-        Instant yesterdayScheduledFireTime = scheduledFireTime.minus(DAILY.getDuration());
+    List<AggregationCommand> processAggregateEmails() {
+        LocalDateTime endTime = LocalDateTime.now();
```

Comment:
Related to my previous comment:
```suggestion
    List<AggregationCommand> processAggregateEmails(LocalDateTime endTime) {
```